### ISSUE TITLE
Adding Linked Text Feature in Merchant Center

### ISSUE THREAD

**Issue by ct-esra** [commercetools/ui-kit#2307]
**The problem:**
As Checkout Anywhere team, we need to provide a page/component where the sellers configure their user agreements (privacy policy, terms of use etc.) in Merchant Center. To do this, sellers should be able to add the agreement text. This text may also include a linked text which redirects buyer to the regarding page as part of legal requirement.
We're not able to provide this with the current capabilities of MC.

**The solution:**
We need a feature that allows sellers to enter a text with a link. As we discussed with Maxim, adding linking functionality on top of the [RichTextInput](https://uikit.commercetools.com/?path=/story/components-inputs--richtextinput) component would work. 

**Note:**
Please note that we don't need other styling features that RichTextInput provides. Because there will be only one type of styling text in our Application so it would be better if you can remove the following features from RichTextInput component to avoid confusion:
- Style
- Bold, Italic, Underline, More styles
- Numbered list, bullet list
Basically, sellers are able to add one type of text and if they want to add a linked text as part of this main text, they should be able to do this.

Please also note that we are still working on the MC Checkout Anywhere Settings Page where sellers can configure their settings. The agreement setting is a part of it.

**Comment by Rhotimee** [comment]
Hello @ct-esra, we will look into this ad and get back to you. Thanks.

**Comment by ct-esra** [comment]
Hello @Rhotimee, thanks!

**Comment by mmaltsev-ct** [comment]
Hey @franzi-t, do you think you can share with us any design concepts for this component, if you already have any, please?

**Comment by franzi-t** [comment]
Hi @mmaltsev-ct, so far, there is no design concept created, but I will prepare something and share it with you.  Thank you.

**Comment by mmaltsev-ct** [comment]
@franzi-t seems like we can close this issue, am I right?
